docs/src/examples/Logger/Basic.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div class="container spaced text-center">
+    Olhe o console do navegador!
+  </div>
+</template>
+
+<script>
+export default {
+  created () {
+    this.$qas.logger.group('Grupo de informações', ['teste'])
+    this.$qas.logger.info('Apenas uma mensagem')
+
+    this.$qas.logger.group('Grupo de informações de erro', ['teste'], true)
```

Comment:
O lance de usar o `true` no terceiro parâmetro pode deixar quem for usar pensando o que é esse `true`. A pessoa tem que ir na função de logger para entender. Será que um objeto de configuração não ficaria mais apropriado? 

Ex:

```
this.$qas.logger.group('Grupo de informações de erro', ['teste'], { error: true })
```